### PR TITLE
Log retries as Verbose when attempting to accept sessions

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Addresses https://github.com/Azure/azure-sdk-for-net/issues/27976

### CODE REVIEW

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/CreateSubscriptionOptions.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ public int MaxDeliveryCount
                 Argument.AssertAtLeast(
                     value,
                     AdministrationClientConstants.MinAllowedMaxDeliveryCount,
-                    nameof(AutoDeleteOnIdle));
+                    nameof(MaxDeliveryCount));
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/CreateSubscriptionOptions.cs:185]: Addresses https://github.com/Azure/azure-sdk-for-net/issues/28289

**Review by jsquire — APPROVED** [review]

**Pull request merged**